app/graphql/types/stat.rb

```diff
@@ -0,0 +1,19 @@
+class Types::Stat < Types::BaseObject
+  description 'The basic stat information.'
+
+  field :id, ID, null: false
+
+  field :profile, Types::Profile,
+    null: false,
+    description: 'The profile related to the user for this stat.',
+    method: :user
+
+  field :data, GraphQL::Types::JSON,
```

Comment:
Big JSON blobs are bad GraphQL practice, so I think we should create separate types defining the schema for each stat type and implementing a Stat interface like we do for Media

Reply:
fineeeeeeeeeee. lol